app/models/redhat_access/telemetry_configuration.rb

```diff
@@ -0,0 +1,14 @@
+module RedhatAccess
+  class TelemetryConfiguration < ActiveRecord::Base
```

Comment:
I suppose we can remove this model once we move to certificate authentication on the cloud.
@lphiri do you know about a schedule for this?